### PR TITLE
feat: overview for verifiable document

### PR DESCRIPTION
**Pull request by yehjxraymond**

### CODE REVIEW

File: docs/verifiable-document/overview.md [hunk 1]
```diff
@@ -4,8 +4,37 @@ title: Verifiable Document Overview
 sidebar_label: Overview
 ---
 
-Overview
+Verifiable documents forms the core of the OpenAttestation(OA) Framework. In this quick start guide, you will be deploying your first verifiable document.
```
Comment (Nebulis) [inline docs/verifiable-document/overview.md]: forms => form

File: docs/verifiable-document/overview.md [hunk 2]
```diff
@@ -4,8 +4,37 @@ title: Verifiable Document Overview
...
 
-## Header
+## Goal
 
-TBD
+By the end of this tutorial you would be able to create your "Certificate of Completion" that is valid on any OA Viewer connected to the Ethereum ropsten network. From there, you may:
+
+1. Change the data structure of the document to suite your needs
+1. Change the rendering of the document to reflect your document styles
+1. Change the backend to use Ethereum main network
```
Comment (Nebulis) [inline docs/verifiable-document/overview.md]: not sure to understand this one ?

File: docs/verifiable-document/overview.md [hunk 3]
```diff
@@ -4,8 +4,37 @@ title: Verifiable Document Overview
...
+
+## Overview of Components
+
+![Overview of Components](/docs/verifiable-document/overview/overview.png)
```
Comment (Nebulis) [inline docs/verifiable-document/overview.md]: In the file
Embeded => Embedded

File: docs/verifiable-document/overview.md [hunk 4]
```diff
@@ -4,8 +4,37 @@ title: Verifiable Document Overview
...
+
+### Document Store Smart Contract
+
+When a document is being issued, a proof of the issuance is store onto the Ethereum blockchain. A smart contract is used to provide a globally consistent record for anyone to query a given document's issuance status.
+
+### DNS Records
+
+A domain is required to issue an OA file. A DNS record will be inserted to the DNS to assert the identity of the OA document creator.
```
Comment (Nebulis) [inline docs/verifiable-document/overview.md]: I dunno if must be is better than will be in that case ?

will be make it sounds we will do it
must be makes it sounds it's mandatory

I leave it to you

File: docs/verifiable-document/overview.md [hunk 5]
```diff
@@ -4,8 +4,37 @@ title: Verifiable Document Overview
...
+
+### Verifiable Document File
+
+Machine-readable data of the document is stored in a .json file. On top of the data, these files also contain information such as:
+
+- claim of issuer's identity
+- document rendering information
+- document store smart contract
+- cryptographic setup to enable selective privacy within the document
```
Comment (Nebulis) [inline docs/verifiable-document/overview.md]: I'm not sure this sentence is simple enough to explain what's it's about. I think we could do a quick test like sending the sentence to some ppl and ask them what they understand about

File: docs/verifiable-document/overview.md (hunk 2 above)
Comment (yehjxraymond) [reply docs/verifiable-document/overview.md]: Added context in the sentence before

File: docs/verifiable-document/overview.md (hunk 5 above)
Comment (yehjxraymond) [reply docs/verifiable-document/overview.md]: Removed for simplicity sake. Developers seeing this for the first time dont have to know that now. Can move that to advanced topics.

**Pull request merged**